src/components/MyKiva/BadgeCompleted.vue

```diff
@@ -99,14 +99,14 @@ const { getTierBadgeDataByLevel, getEarnedBadgeExplanation } = useBadgeData();
 
 const badgeData = computed(() => getTierBadgeDataByLevel(badge.value, tier.value?.level));
 
-const isPublic = computed(() => lender.value?.public && lender.value?.publicName);
+const isPublic = computed(() => lender.value?.public && lender.value?.name);
```

Comment:
What issue is this resolving? Please make sure we create tickets for issues and link in PR descriptions, thanks!